bigbluebutton-tests/playwright/learningdashboard/learningdashboard.spec.ts

```
@@ -0,0 +1,40 @@
+  });
+
+  test('Check message', async () => {
+    await learningDashboard.writeOnPublicChat();
+  });
+
+  test('User Time On Meeting', async () => {
+    await learningDashboard.userTimeOnMeeting();
+  });
+
+  // eslint-disable-next-line no-empty-pattern
+  test('Polls', { tag: '@flaky' }, async ({}, testInfo) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)